web3.md

```diff
@@ -1361,17 +1370,26 @@ Transaction Execution
 
     rule <k> #eth_estimateGas
           => #pushNetworkState
-          ~> #loadTx J
+          ~> #loadTx #parseHexWord( #getString("from", J) ) J
           ~> #eth_estimateGas_finalize GUSED
          ...
          </k>
          <params> [ ({ _ } #as J), TAG, .JSONs ] </params>
          <gasUsed>  GUSED  </gasUsed>
       requires isString(#getJSON("from", J) )
 
-    rule <k> #eth_estimateGas => #rpcResponseError(-32028, "Method 'eth_estimateGas' has invalid arguments") ...  </k>
+    rule <k> #eth_estimateGas
+          => #pushNetworkState
+          ~> #loadTx {keys_list(ACCTS)[0]}:>Int J
+          ~> #eth_estimateGas_finalize GUSED
+         ...
+         </k>
          <params> [ ({ _ } #as J), TAG, .JSONs ] </params>
-      requires notBool isString( #getJSON("from", J) )
+         <gasUsed>  GUSED  </gasUsed>
+         <accountKeys> ACCTS </accountKeys>
+      requires notBool isString(#getJSON("from", J) )
```

Comment:
Could we also just have a rule which does something like this:

```
rule <k> #eth_estimateGas ... </k>
      <params> [ { _ , (.JSONs => "from": {keys_list(ACCTS)[0]}:>Int , .JSONs) } #as J ], TAG, .JSONs </params>
  requires notBool isString(#getJSON("from", J))
```

that way we only have one rule describing the semantics of `#eth_estimateGas` instead of duplicating the logic?